docs/07Examples/19-EEGProjects/20-Motor_Imagery.md

```diff
@@ -0,0 +1,152 @@
+
+## ACQUISITION OF EEG DATA AND MARKERS
+
+Plug in the RFduino dongle, connect electrodes to the cyton board pins. Wear the EEG headset and finally connect the ear clip to SRB. Open the OpenBCI GUI, select the appropriate port number and start streaming data from the Cyton board. Go to the networking tab and select the LSL protocol. Select “TIME-SERIES” data type and start streaming.
+
+![Hardware setting in the GUI](assets/TutorialImages/Hardware_Setting_in_the_GUI.png)
+
+
+![The OpenBCI GUI](assets/TutorialImages/The_OpenBCI_GUI.png)
+
+
+Before we start classifying the Motor Imagery data, we need to calibrate the system.
+
+## RECORDING CALIBRATION DATA
+
+The NeuroPype pipeline is doing a great job, but wouldn’t it be nice if we didn’t have to recollect the calibration data each time we run the pipeline? It’s often more convenient to record calibration data into a file in the first session, and load that file every time we run our pipeline. For this, we need to use the Inject Calibration Data node, which has a second input port where one can pipe a calibration recording (which we import here using Import XDF).
+Start the Lab recorder and find the OpenBCI EEG stream in the window. Now run the python script motorimg_calibrate.py found in the extras folder in your Neuropype installation folder. Now update the streams in the lab recorder. You should now see MotorImag-Markers and obci_eeg1 stream along with your computer name.
+
+![Lab Recorder](assets/TutorialImages/Lab_Recorder.png)
+
+The python script along with OpenBCI, lab recorder is used to record calibration data. The script sends markers matching what the person is imagining that is 'Left' or 'Right' and instructs the user when to imagine that movement which will be stored in the .xdf file along with the EEG data.
+Run the python script and start recording the OpenBCI stream and markers stream using the lab recorder. Follow the instructions shown on the window: when the window shows ‘R’  imagine moving your right arm, and when it shows ‘L’   imagine moving your left arm.  It takes about half a second for a person to read the instruction and begin imagining the movement, and he/she will finish about 3 seconds later and get ready for the next trial. This is why the segment time limits in segmentation node are set to (0.5,3.5).
```

Comment:
Before: "segment time limits in segmentation node are set to (0.5,3.5)."
After Change: segment time limits in the segmentation node are set to (0.5,3.5).